content/en/dashboards/screenboards.md

```diff
@@ -21,29 +21,29 @@ Screenboards are dashboards with free-form layouts which can include a variety o
 
 ## Global time selector
 
-To use the screenboard global time selector, at least one time-based widget must be set to use `Global Time`. Make the selection in a widget's editor under **Set display preferences**, or add a widget (global time is the default time setting).
+To use the screenboard global time selector, at least one time-based widget must be set to use `Global Time`. Make the selection in a widget’s editor under **Set display preferences**, or add a widget (global time is the default time setting).
 
-The global time selector sets the same timeframe for all widgets using the `Global Time` option on the same screenboard. You can select a moving window in the past (`The Past Hour`, `The Past Day`, etc.) or a fixed period with the `Select Range` option. If a moving window is chosen, the widgets are updated to move along with the time window.
+The global time selector sets the same time frame for all widgets using the `Global Time` option on the same screenboard. You can select a moving window in the past (`Past 1 Hour`, `Past 1 Day`, etc.) or a fixed period with the `Select from calendar…` option or by [entering a custom time frame][1]. If a moving window is chosen, the widgets are updated to move along with the time window.
```

Comment:
Note about `timeframe` => `time frame`. It's more common in the app for it to be styled as two words, so I figured we should reflect that here...